### ISSUE TITLE
Invalid position given by 'W033'

### ISSUE THREAD

**Issue by ETGgames** [AtomLinter/linter-jshint#469]
JSHint returned a point that did not exist in the document being edited.
Rule: `W033`
Requested point: 7:6

Debug information:
```
Atom version: 1.18.0
linter-jshint version: v3.1.5
JSHint version: jshint v2.9.5
Hours since last Atom restart: 0
Platform: darwin
Current file's scopes: [
  "source.js",
  "meta.method-call.js",
  "meta.arguments.js",
  "punctuation.definition.arguments.end.bracket.round.js"
]
linter-jshint configuration: {
  "executablePath": "",
  "lintInlineJavaScript": false,
  "disableWhenNoJshintrcFileInPath": false,
  "scopes": [
    "source.js",
    "source.js-semantic"
  ],
  "jshintFileName": ".jshintrc",
  "jshintignoreFilename": ".jshintignore"
}
```

**Comment by ETGgames** [comment]
please dont flag as duplicate, the link you provided does not give a solution.

**Comment by Arcanemagus** [comment]
I never said any of this gave a _solution_, the problem is that JSHint is (and always has been) an incredibly buggy linter. If you are looking for a reliable linter I would _highly_ recommend switching to ESLint, it is the gold standard of all the dozens of linters that I have seen and lucky enough for us it happens to be a linter for Javascript 😉.

Closing as a duplicate of #422, please follow the JSHint issue linked there for more information. If you want a solution you can help them figure out a fix for this issue there.
